### PR TITLE
Fix: Correct Dart Get Session By ID snippet

### PR DESCRIPTION
**Pull request by adityaoberai**
## Description  
Corrects the Get Session By ID snippet for the Dart server-side SDK

## Testing  
Followed steps for testing as specified in the Contributing Guide

## Checklist  
- [x] Followed the [contribution guidelines](https://github.com/2002Bishwajeet/awesome-appwrite-snippets/blob/master/CONTRIBUTING.md)  
- [x] The code is commented properly to give appropriate context.

### CODE REVIEW

File: snippets/dart/account.code-snippets [hunk 1]
```diff
@@ -47,7 +47,7 @@
       "scope": "dart",
       "prefix": ["sGetSessionId", "sGetSessionbyId"],
       "body": [
-        "final ${1:response} = await account.getSesssion(sessionId: '${1:sessionId}');"
+        "final ${1:response} = await account.getSession(sessionId: '${1:sessionId}');"
```
Comment (2002Bishwajeet) [inline snippets/dart/account.code-snippets:50]:
```suggestion
        "final ${1:response} = await account.getSession(sessionId: '${2:sessionId}');"
```

File: snippets/dart/account.code-snippets [hunk 2]
```diff
@@ -153,4 +153,4 @@
       "description": "Use this endpoint to complete the user phone verification process. Use the userId and secret that were sent to your user's phone number to verify the user email ownership. If confirmed this route will return a 200 status code."
     }
   }
-  
\ No newline at end of file
+
```
Comment (2002Bishwajeet) [inline snippets/dart/account.code-snippets:156]:
```suggestion
  
```

**Comment by adityaoberai** [comment]
@2002Bishwajeet made the changes

**Review by 2002Bishwajeet — APPROVED** [review]
LGTM!

**Pull request merged**